ts-graphql-yoga/index.ts

```diff
@@ -0,0 +1,36 @@
+import * as express from "express";
+import { createServer } from "@graphql-yoga/node";
+
+// Create a GraphQL Yoga Server with a simple schema
+const graphQLServer = createServer({
+  schema: {
+    typeDefs: /* GraphQL */ `
+      type Query {
+        hello: String
+      }
+    `,
+    resolvers: {
+      Query: {
+        hello: () => 'Hello from Klotho!',
+      },
+    },
+  },
+});
+
+// Create an Express app that will be used to serve a GraphQL Yoga server as middleware
```

Comment:
The other sample-apps don't have these kinds of comments, right? I like them a lot! (This one and the similar ones below)

Reply:
I think there are a few comments sprinkled throughout the samples, mostly to help with ambiguity. I don't think other samples have the same type of explanatory comments.